doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,909 @@
+  * The :ref:`ug_matter_device_low_power_configuration` page with information about sleepy active threshold parameter configuration.
+  * The default number of user RTC channels on the nRF5340 SoC's network core from 2 to 3 (the platform default) to fix the CSL transmitter feature on Matter over Thread devices acting as Thread routers.
+  * :ref:`ug_matter_hw_requirements` with updated memory requirement values valid for the |NCS| v2.4.0.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.1.0.1`` tag.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+ * ``SLEEPY_ACTIVE_THRESHOLD`` parameter that makes the Matter sleepy device stay awake for a specified amount of time after network activity.
+
+* Updated:
+
+  * The factory data generation script with the feature for generating the onboarding code.
+    You can now use the factory data script to generate a manual pairing code and a QR Code that are required to commission a Matter-enabled device over Bluetooth LE.
+    Generated onboarding codes should be put on the device's package or on the device itself.
+    For details, see the Generating onboarding codes section on the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
+  * The Basic Information cluster with device finish and device color attributes and added the related entries in factory data set.
+
+Thread
+------
+
+|no_changes_yet_note|
+
+See `Thread samples`_ for the list of changes for the Thread samples.
+
+Zigbee
+------
+
+* Fixed an issue where device is not fully operational in the distributed network by adding call to the :c:func:`zb_enable_distributed` function for Zigbee Router role by default.
+
+See `Zigbee samples`_ for the list of changes for the Zigbee samples.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added:
+
+  * Support for bigger payload size.
+    ESB supports a payload with a size of 64 bytes or more.
+  * The :ref:`esb_fast_ramp_up` feature that reduces radio ramp-up delay from 130 µs to 40 µs.
+  * The :kconfig:option:`CONFIG_ESB_NEVER_DISABLE_TX` Kconfig option as an experimental feature that enables the radio peripheral to remain in TXIDLE state instead of TXDISABLE when transmission is pending.
+    For more information, see :ref:`esb_never_disable_tx`.
+
+* Updated:
+
+  * The number of PPI/DPPI channels used from three to six.
+  * Events 6 and 7 from the EGU0 instance by assigning them to the ESB module.
+  * The type parameter of the :c:func:`esb_set_tx_power` function to ``int8_t``.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+|no_changes_yet_note|
+
+Wi-Fi
```

Comment:
I am assuming that this needs to be updated based on board renaming

Reply:
The change is done in this PR and will be integrated here once the PR is merged:
https://github.com/nrfconnect/sdk-nrf/pull/11289/files#diff-17b816d17cd5f06420ac3e251212a721de6dd397df809411ea0b0cf974a877d2

Reply:
Pls add the below update here -

Added
•	Listen interval based power-save – refactored existing power-save configurations into one.
•	Support for configuring antenna gain and band-edge power.
•	Support for nRF700x ns build.
•	Support to interop with CMW500 configured as Wi-Fi Access Point.
•	Support to configure regulatory domain.
•	Support for raw scan results.
Updated:
•	TWT features with improvements (units for interval changed from milliseconds to microseconds).

Reply:
Please add the below updates here -
Added
•	Listen interval based power-save – refactored existing power-save configurations into one.
•	Support for configuring antenna gain and band-edge power.
•	Support for nRF700x ns build.
•	Support to interop with CMW500 configured as Wi-Fi Access Point.
•	Support to configure regulatory domain.
•	Support for raw scan results.
Updated:
•	TWT features with improvements (units for interval changed from milliseconds to microseconds).

Reply:
Added